### PR TITLE
Refactored

### PR DESCRIPTION
**Pull request by ethanysh**
yet to be commented

### CODE REVIEW

File: scraper_atlanta.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+def scraper_atlanta(chrome_path: str,
+                    url='https://aca3.accela.com/Atlanta_Ga/Welcome.aspx',
+                    days_to_scrape=1) -> dict:
+    """
+    The actual scraping process.
+    :param chrome_path: the path in the system where `chromedriver` is stored
+    :param url: the url of the website to be scraped
+    :param days_to_scrape: days prior to today to start scraping. 1 means from yesterday to today, 2 days in total.
+           The value is defaulted to 2 days.
```
Comment (ycshao) [inline scraper_atlanta.py]:
```suggestion
           The value is defaulted to 1 days.
```

File: scraper_atlanta.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    element.send_keys((datetime.today()
+                       - timedelta(days=days_to_scrape)).strftime('%m/%d/%Y'))  # enter `date_from`
+    driver.find_element_by_xpath(Element.SEARCH_BUTTON_XPATH.value).click()  # click the `search` button
+
+    table_rows = get_table(driver, Element.TABLE_ID.value)
+    headers = get_headers(table_rows)
+    permits = dict()
+
+    # determine if there are more than 1 page of the results
+    if len(table_rows) < TableHTML.HEADER_AND_ABOVE.value \
```
Comment (ycshao) [inline scraper_atlanta.py]: Better way is to use the existence of the elements of `aca_pagination` to determine if there is only one row or not I think.

File: scraper_atlanta.py [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+
+class TableHTML(Enum):
+    """
+    The class contains variables that describe the structure of the table to be scraped
+    """
+    HEADER_AND_ABOVE = 3  # table header and content above has 3 `tr` tags
+    PAGE_NAV = 2  # page navigation bar has 2 `tr` tags
+    CONTENT_AND_PAGE_NAV = 12  # maximum table content plus page navigation bar has 12 `tr` tags
+    HEADER_ROW_POSITION = 2  # the 3rd 'tr' tag in table tag
```
Comment (ycshao) [inline scraper_atlanta.py:17]: I see header row is always the second `tr` element in `tbody` tag. `ACA_TabRow_Header ACA_TabRow_Header_FontSize` is the class name.

Comment (ethanysh) [reply scraper_atlanta.py:17]: There's another `tr` nested in the first `tr` actually.
I tried to use the class name for the header at the beginning, it says selenium doesn't support some thing called `compound class name` or something like that. I guess there's an space in the class name. And oftenly, i cannot locate the element by `xpath`  in selenium. That's why I used the absolute position to locate the element.

File: scraper_atlanta.py (hunk 2 above)
Comment (ethanysh) [reply scraper_atlanta.py]: fixed

**Review by ycshao — APPROVED** [review]

**Pull request merged**